src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    const { type } = action;
+
+    switch (type) {
+      case 'addProperties':
+        const { extraData } = action;
 
+        Object.assign(state, extraData);
+        break;
```

Comment:
consider adding empty line after each break statement - it is easier to read

Reply:
I have checked the code again, also took a look at other students ones, now I understand what was wrong. Hopefully it matches everything now